From 4ce7d0d535b9542d4556c91e3416c2a35fc48a64 Mon Sep 17 00:00:00 2001
From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Thu, 20 Aug 2020 20:15:34 +0200
Subject: Enable darken websites checkbox in themes

Unexpire #darken-websites-checkbox-in-themes-setting

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 chrome/browser/flag-metadata.json                           | 6 +++---
 .../Enable-darken-websites-checkbox-in-themes.inc           | 5 +++++
 2 files changed, 8 insertions(+), 3 deletions(-)
 create mode 100644 cromite_flags/components/content_settings/core/common/features_cc/Enable-darken-websites-checkbox-in-themes.inc

diff --git a/chrome/browser/flag-metadata.json b/chrome/browser/flag-metadata.json
index 0308b9bd2f830..998f4c4c13fcf 100644
--- a/chrome/browser/flag-metadata.json
+++ b/chrome/browser/flag-metadata.json
@@ -2437,9 +2437,9 @@
     "expiry_milestone": 160
   },
   {
-    "name": "darken-websites-checkbox-in-themes-setting",
-    "owners": [ "nemco@google.com", "wenyufu@google.com", "twellington@chromium.org" ],
-    "expiry_milestone": 160
+    "name": "darken-websites-checkbox-in-themes-setting", // this is always enabled in Bromite
+    "owners": [ "nemco@google.com", "wenyufu@google.com", "twellington@chromium.org" ], // -
+    "expiry_milestone": -1
   },
   {
     "name": "data-controls-search-with",
diff --git a/cromite_flags/components/content_settings/core/common/features_cc/Enable-darken-websites-checkbox-in-themes.inc b/cromite_flags/components/content_settings/core/common/features_cc/Enable-darken-websites-checkbox-in-themes.inc
new file mode 100644
index 0000000000000..e4c1a035ab238
--- /dev/null
+++ b/cromite_flags/components/content_settings/core/common/features_cc/Enable-darken-websites-checkbox-in-themes.inc
@@ -0,0 +1,5 @@
+#if BUILDFLAG(IS_ANDROID)
+
+SET_CROMITE_FEATURE_ENABLED(kDarkenWebsitesCheckboxInThemesSetting);
+
+#endif  // BUILDFLAG(IS_ANDROID)
-- 
2.55.0

